gadgets/Makefile

```diff
@@ -2,6 +2,7 @@ SHELL := /bin/bash
 
 GADGET_TAG ?= $(shell ../tools/image-tag branch)
 GADGET_REPOSITORY ?= ghcr.io/inspektor-gadget/gadget
+GADGET_VERIFY_IMAGE ?= true
```

Comment:
I'm wondering if we want to have a specific flag or a generic one as I proposed on https://github.com/inspektor-gadget/inspektor-gadget/pull/3025/commits/9c60218bb1b9b8b7eab36f642e5717e83d5ca5f7